src/modules/message/components/signMessageView/index.js

```diff
@@ -18,17 +18,23 @@ const signMessageView = ({ account, t, history, signMessage }) => {
   ] = useCurrentAccount();
 
   const onMultiStepChange = useCallback(({ step: { current } }) => {
+    console.log('....', current);
     setMultiStepPosition(current);
   }, []);
 
   return (
     <Dialog hasClose className={styles.wrapper}>
       <Box>
-        {multiStepPosition !== 1 && (
+        {multiStepPosition === 0 && (
           <BoxHeader>
             <h1>{t('Sign message')}</h1>
           </BoxHeader>
         )}
+        {multiStepPosition === 2 && (
+          <BoxHeader>
+            <h1>{t('Signed message')}</h1>
+          </BoxHeader>
+        )}
```

Comment:
please avoid duplication
```suggestion
        {(multiStepPosition === 0 || multiStepPosition === 2) && (
          <BoxHeader>
            <h1>{t('Sign message')}</h1>
          </BoxHeader>
        )}
       
```

Reply:
better option
```suggestion
const hasHeader = multiStepPosition === 0 || multiStepPosition === 2
        {hasHeader && (
          <BoxHeader>
            <h1>{t('Sign message')}</h1>
          </BoxHeader>
        )}
       
```

Reply:
The header needs to change based on the state, once the message is signed it shouldn't be called `Sign message`? so it was updated to `Signed message`.

Reply:
we can have the condition only for text
```suggestion
        {multiStepPosition !== 1 && (
          <BoxHeader>
            <h1>{multiStepPosition === 2 ? t('Signed message')  : t('Sign message')}</h1>
          </BoxHeader>
        )}
       
```